src/components/Modal/readme.md

```diff
@@ -708,3 +708,276 @@ class FormModal extends React.Component {
     </GlobalHeader>
 </div>
 ```
+
+##### make reservation modal
+
+```js
+import React from 'react';
+import { Modal, Button, Input, TimePicker, DatePicker, Textarea, Select } from 'react-rainbow-components';
+import { Field, reduxForm } from 'redux-form';
+import styled from 'styled-components';
+
+const Content = styled.div.attrs( props => ({
+    imgWidth: '328px'
```

Comment:
since this value is fixed and always the same you can put it in a const like:
```
const imgWidth = '328px';

const Content = styled.div`
    div:nth-child(2) img {
        min-width: ${imgWidth};
    }
    ...
`
```